authnz/src/main/java/io/stargate/auth/PlainTextTokenSaslNegotiator.java

```diff
@@ -35,7 +35,7 @@
 
   protected final AuthenticationService authentication;
   private final Authenticator.SaslNegotiator wrapped;
-  protected StoredCredentials storedCredentials;
+  protected AuthenticationPrincipal authenticationPrincipal;
   protected final String tokenUsername;
   protected final int tokenMaxLength;
   private static final boolean USE_TRANSITIONAL_AUTH =
```

Comment:
`USE_TRANSITIONAL_AUTH` does not seem to be used any more.

Reply:
removed